src/app/user-list/user-list.component.ts

```diff
@@ -1,15 +1,16 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit } from "@angular/core";
+import { UserService } from "../user.service";
 
 @Component({
-  selector: 'app-user-list',
-  templateUrl: './user-list.component.html',
-  styleUrls: ['./user-list.component.scss']
+  selector: "app-user-list",
+  templateUrl: "./user-list.component.html",
+  styleUrls: ["./user-list.component.scss"]
 })
 export class UserListComponent implements OnInit {
-
-  constructor() { }
+  userList;
+  constructor(private userService: UserService) {}
 
   ngOnInit() {
+    this.userService.getUsers().subscribe(users => (this.userList = users));
```

Comment:
you don't need parentheses here "(this.userList = users)"

Reply:
its not me, auto-prettier make it)